java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -880,6 +880,22 @@ public void expireAt_pexpireAt_and_ttl_with_non_existing_key(BaseClient client)
         assertEquals(-2L, client.ttl(key).get());
     }
 
+    @SneakyThrows
+    @ParameterizedTest
+    @MethodSource("getClients")
+    public void persist_on_existing_key(BaseClient client) {
```

Comment:
```suggestion
    public void persist_on_existing_non_existing_key(BaseClient client) {
```

and check also before SET

Reply:
Update for both